lib/resolve-package-path.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var findPackagePath = require('./find-package-path');
+var heimdall = require('heimdalljs');
+var path = require('path');
+var getRealFilePath = require('./get-real-file-path');
+/*
+ * Canonicalize a file path from name, dir into a real path. It's possible the path
+ * does not actually resolve to a real file, in which case null is returned.
+ */
+module.exports = function resolvePackagePath(realFilePathCache, name, dir) {
+  var h = heimdall.start('hashForDep:resolvePackagePath');
+
+  try {
+    if (name === './' || name.charAt(0) === '/') {
```

Comment:
Do we normalize to posix paths? How does this work on windows?

Reply:
I don't think so, don't know for sure. If it can be done fast and as few times as possible, great. If it has to be done for every path during every resolution, leaving them as is seems faster. But I'm willing to be convinced I'm wrong. :-)